geode-core/src/main/java/org/apache/geode/internal/security/SecurityServiceFactory.java

```diff
@@ -28,6 +28,8 @@
 import org.apache.geode.security.SecurityManager;
 
 public class SecurityServiceFactory {
+  public static final String SECURITY_SERVICE_SYSTEM_PROPERTY =
+      "org.apache.geode.internal.security.SecurityService";
```

Comment:
Is this new property? Does it impact current behavior?
When we introduce system property we use SystemPropertyHelper class to define the system property...Is this not a requirement here...

Reply:
This is only for testing